### PR TITLE
Гречухин - Лабораторная работа #0

### PR DESCRIPTION
**Pull request by nik709**
@PovelikinRostislav @might52

### CODE REVIEW

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/GrechukhinNikitaTest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    @Test
+    public void canCreateWithStringParameters() {
+        var complexNumber = new ComplexNumber("2.11", "9.4");
+        assertNotNull(complexNumber);
+    }
+
+    @Test
+    public void canGetRealPart() {
+        var complexNumber = new ComplexNumber(10.2, 93.4);
+        assertEquals(10.2, complexNumber.getReal(), delta);
```
Comment (rpovelik) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/GrechukhinNikitaTest.java]: Expected value should not be a magical constant, create a dedicated variable, please

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/GrechukhinNikitaTest.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    }
+
+    @Test
+    public void canGetImaginaryPart() {
+        var complexNumber = new ComplexNumber(0.0, 0.1);
+        assertEquals(0.1, complexNumber.getImaginary(), delta);
```
Comment (rpovelik) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/GrechukhinNikitaTest.java]: The same as on line 33

**Review by rpovelik — CHANGES_REQUESTED** [review]
Reviewed

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/GrechukhinNikitaTest.java (hunk 1 above)
Comment (nik709) [reply code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/GrechukhinNikitaTest.java]: Fixed

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/GrechukhinNikitaTest.java (hunk 2 above)
Comment (nik709) [reply code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/GrechukhinNikitaTest.java]: Fixed

**Comment by rpovelik** [comment]
Looks good :+1

**Comment by might52** [comment]
good 👍

**Comment by nik709** [comment]
@kirill-korniakov

**Pull request merged**